wrappers/services_test.go

```diff
@@ -66,6 +66,38 @@ type servicesTestSuite struct {
 
 var _ = Suite(&servicesTestSuite{})
 
+type serviceState struct {
+	activeState   string
+	unitFileState string
+}
+
+// If `cmd` is the command issued by systemd.Status(), this function returns
+// the output to be produced by the command so that the queried services will
+// appear having the ActiveState and UnitFileState according to the data
+// passed in the `states` map.
+func mockSystemCtlShow(cmd []string, states map[string]serviceState) []byte {
```

Comment:
(nitpick) This looks very similar to the mock used in `overlord/servicestate/service_control_test.go` earlier in this PR. Which makes me wonder if we should have something like `systemd.MockAllUnitsActiveOutput` or something. But probably followup material (if at all) as this will probably need some discussion about the right name for this new helper.

Reply:
Yes, indeed. There's also the question of the file in which this function should reside. If it is in `systemd.go`, will it end up making the snapd binary bigger? Or is the go compiler smar enough not to include functions that are unused?

Reply:
The go compiler should be smart enough to trim out unused code from the final binary and even if it didn't honestly this one function is not going to significantly increase the size of the binary

Reply:
(that is to say I agree with @mvo5's suggestion to combine these helpers)

Reply:
Done, but I can't really say that I like it. :-) We end up polluting the production namespace with code and classes that are useful for testing only (like the `ServiceState` struct); what would you say about moving this to the `systemd/export_test.go` file, or to some other test support package, if we have one?